Wox.Plugin.Runner/Services/ExplorerPathsService.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+
+namespace Wox.Plugin.Runner
+{
+    class ExplorerPathsService
+    {
+        public static List<string> GetOpenExplorerPaths()
+        {
+            var results = new List<string>();
+            if (OperatingSystem.IsWindows())
```

Comment:
Just saw this, is this referring to OS type? Flow is windows only so not sure if this check is necessary.